server.py

```diff
@@ -106,6 +106,25 @@ def shutdown_screenly():
     sh.sudo('shutdown', 'now', _bg=True)
 
 
+def append_usb_assets(mountpoint):
```

Comment:
I don't know celery but do we need `@celery.task` here as well?

Reply:
You're right. I lost it when I was solving merge conflicts. Thanks!